unicodetools/data/ucd/dev/BidiTest.txt

```diff
@@ -66,7 +66,9 @@
 #   @Reorder:    3 1
 #
 # A data line has the following format:
+
 # <input> ; <bitset>
+
```

Comment:
Blank lines in the headers should still start with `#`.
Also, why is this bundled with the date changes?

Reply:
This commit is the output (verbatim) from a MakeUnicodeFiles run after changing the copyright year to 2024.

Reply:
If the generator needs to be changed, can we do that in a separate PR please?